exporter/opentelemetry-exporter-zipkin/tests/encoder/test_v1_json.py

```diff
@@ -0,0 +1,252 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+from opentelemetry import trace as trace_api
+from opentelemetry.exporter.zipkin.encoder import NAME_KEY, VERSION_KEY
+from opentelemetry.exporter.zipkin.encoder.v1.json import JsonV1Encoder
+from opentelemetry.exporter.zipkin.node_endpoint import NodeEndpoint
+from opentelemetry.sdk import trace
+from opentelemetry.trace import TraceFlags
+
+from .common_tests import CommonEncoderTestCases
+
+
+# pylint: disable=protected-access
+class TestV1JsonEncoder(CommonEncoderTestCases.CommonJsonEncoderTest):
+    @staticmethod
+    def get_encoder(*args, **kwargs) -> JsonV1Encoder:
+        return JsonV1Encoder(*args, **kwargs)
```

Comment:
Nit: this method does nothing but instantiate an object with exactly the same arguments received, is it really necessary?

Reply:
It is due to the abstraction with `CommonEncoderTestCases` which defines the abstract method `get_encoder()` and also invokes it in several common test methods. Each concrete encoder test suite then defines the `get_encoder()` to return its specific encoder object type.